.github/workflows/cd-deploy-nodes-gcp.yml

```diff
@@ -48,32 +48,12 @@ on:
   #    - 'docker/**'
   #    - '.dockerignore'
   #    - '.github/workflows/continous-delivery.yml'
-  #    - '.github/workflows/build-docker-image.yml'
-
-  # Only runs the Docker image tests, doesn't deploy any instances
-  pull_request:
```

Comment:
The Docker release image checks need to run on PRs so that PRs don't break the release images.

Reply:
Reverted